blaze-client/src/main/scala/org/http4s/client/blaze/bits.scala

```diff
@@ -23,31 +23,21 @@ private[blaze] object bits {
 
   val ClientTickWheel = new TickWheelExecutor()
 
-
-
   def getExecutor(config: BlazeClientConfig): (ExecutorService, Task[Unit]) = config.customExecutor match {
     case Some(exec) => (exec, Task.now(()))
     case None =>
       val exec = DefaultExecutor.newClientDefaultExecutorService("blaze-client")
       (exec, Task { exec.shutdown() })
   }
 
-  /** The sslContext which will generate SSL engines for the pipeline
-    * Override to provide more specific SSL managers */
-  lazy val sslContext = defaultTrustManagerSSLContext()
-
-  private class DefaultTrustManager extends X509TrustManager {
-    def getAcceptedIssuers(): Array[X509Certificate] =  new Array[java.security.cert.X509Certificate](0)
-    def checkClientTrusted(certs: Array[X509Certificate], authType: String): Unit = {}
-    def checkServerTrusted(certs: Array[X509Certificate], authType: String): Unit = {}
-  }
-
-  private def defaultTrustManagerSSLContext(): SSLContext = try {
+  lazy val TrustingSslContext: SSLContext = {
```

Comment:
Since, this is in a different location then the Docs which are in BlazeClientConfig could we a add a short doc here explaining  "trusts all certificates and disables endpoint identification"